### PR TITLE
Reduce children name questions to 1 question to avoid developer confu…

### LINKED ISSUE #781: Can we have 1 children[i].name_field() question?
It is difficult to understand and then overwrite these 2 so generic questions - I found it very confusing.
I didn't realize there are **2** questions that need to be used or overwritten. 
And the questions as they are in the code are difficult to understand. 
#780

### PR DESCRIPTION
**Pull request by plocket**
…sion about how to override questions. Closes #781

### CODE REVIEW

File: docassemble/AssemblyLine/data/questions/ql_baseline.yml [hunk 1]
```diff
@@ -815,31 +815,17 @@ validation code: |
...
-# Customize this question if you know whose children they are in your interview!
-# this is just a very generic way to ask.
-id: children names without birthdate
 sets:
   - children[i].name.first
   - children[i].name.last
   - children[i].name.middle
   - children[i].name.suffix
 question: |
+  % if hasattr(children, 'ask_number') and children.ask_number and children.target_number == 1:
```
Comment (nonprofittechy) [inline docassemble/AssemblyLine/data/questions/ql_baseline.yml:824]:
```suggestion
  % if hasattr(children, 'ask_number') and children.ask_number and children.target_number == 1 and i == 0:
```

Comment (plocket) [reply docassemble/AssemblyLine/data/questions/ql_baseline.yml:824]: I'm happy to make the change, but I just want to check on my understanding. If the target number is 1, won't `i == 0` always resolve to `True`?

Comment (nonprofittechy) [reply docassemble/AssemblyLine/data/questions/ql_baseline.yml:824]: no. target number is only used when you use .gather(). If there's an "add" button you can get past it.

**Review by nonprofittechy — APPROVED** [review]
This looks fine. I don't totally agree it's easier to have one question than 2 for customizing purposes, but I think this change doesn't make it a lot harder either.

**Pull request merged**